homeassistant/components/sunsynkweb/const.py

```diff
@@ -0,0 +1,13 @@
+"""Constants for the Sunsynk Inverter Web integration."""
+
+DOMAIN = "sunsynkweb"
+"""stub and constants for api accessm."""
+
+BASE_URL = "https://api.sunsynk.net"
+BASE_API = BASE_URL + "/api/v1"
+BASE_HEADERS = {
+    "accept": "application/json",
+    "content-type": "application/json",
+    "accept-language": "en-US,en;q=0.5",
+    "accept-encoding": "gzip, deflate, br",
+}
```

Comment:
```suggestion
```
appear unused

Reply:
it is used in config_flow

Reply:
oh right, but only the `BASE_URL`. If that's the only place it is used, move it there and the rest is indeed unused (if I'm not mistaken again)